apps/lib/apps.c

```diff
@@ -2252,7 +2252,7 @@ double app_tminterval(int stop, int usertime)
         struct timeval tv;
         gettimeofday(&tv, NULL);
         now = (clock_t)((unsigned long long)tv.tv_sec * CLK_TCK +
-                        (unsigned long long)tv.tv_usec * (1000000 / CLK_TCK)
+                        (unsigned long long)tv.tv_usec * (CLK_TCK / 1000000)
```

Comment:
Oh!!!!!

Reply:
I've noticed that too, but this is still wrong. You will now get 0, you need to remove the ().

Reply:
Please also fix the uses of () for the if above.

Reply:
Thanks @kroeckx.  I just had another look at the definition of `CLK_TCK` on VMS.  It's 100, which had me wonder "what the hell?????"

Reply:
Note that it's only used for -elapsed, so you probably won't notice it's weird behaviour with things like negative time.